openbao-vault/info.json

```diff
@@ -0,0 +1,148 @@
+{
+  "name": "openbao-vault",
+  "version": "1.0.0",
+  "label": "OpenBao Vault",
+  "description": "OpenBao is an identity-based secrets and encryption management system, designed to securely manage, store, and distribute sensitive data such as API keys, passwords, certificates, and more.",
+  "publisher": "Fortinet",
+  "contributor": "",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "tags": [
+    "vault"
+  ],
+  "category": "Vault",
+  "help_online": "",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "ingestion_supported": false,
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "name": "server_url",
+        "description": "Specify the URL of the OpenBao server to which you will connect and perform automated operations.",
+        "type": "text",
+        "visible": true,
+        "editable": true,
+        "required": true,
+        "tooltip": "Specify the URL of the OpenBao server to which you will connect and perform automated operations."
+      },
+      {
+        "title": "Access Token",
+        "name": "token",
+        "description": "Specify the token of the OpenBao server to which you will connect and perform automated operations.",
+        "type": "password",
+        "visible": true,
+        "editable": true,
+        "required": true,
+        "tooltip": "Specify the token of the OpenBao server to which you will connect and perform automated operations."
+      },
+      {
+        "title": "Namespace",
+        "name": "namespace",
+        "description": "Specify the namespace OpenBao server to which you will connect and perform automated operations.",
+        "type": "text",
+        "visible": true,
+        "editable": true,
+        "required": true,
+        "tooltip": "Specify the namespace OpenBao server to which you will connect and perform automated operations."
+      },
+      {
+        "title": "Secret Engine Name",
+        "name": "secret_name",
+        "description": "Specify the name of the secret engine based on which you will connect and perform automated operations.",
+        "type": "text",
+        "visible": true,
+        "editable": true,
+        "required": true,
+        "tooltip": "Specify the name of the secret engine based on which you will connect and perform automated operations."
+      },
+      {
+        "title": "Verify SSL",
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True.",
+        "name": "verify_ssl",
+        "type": "checkbox",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "value": true,
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_credentials",
+      "visible": false,
```

Comment:
Why is this action marked as visible = false? Is it used internally or was it marked like this by mistake?